05week/transitions-and-transformations/css/style.css

```diff
@@ -0,0 +1,93 @@
+.box{
+  display: flex;
+  height:50px;
+  width:50px;
+  border-width: 3px;
+  border-color: black;
+  border-style: solid;
+  transition: width 2s, height 2s, background-color 2s, transform 2s;
+   margin 3s;
+
+}
+/* Create a webpage that consists of 12 different colored squares with side lengths
+of 50px.
+The squares should start in a vertical line in the middle of the page.
+When you hover over the squares, every other square should grow to be twice as large.
+The other squares should shrink to half their original size.
+Then the larger squares should slowly slide to the right side of the page, rotating
+clockwise as they move and remaining in a vertical line.
+Then the smaller squares should quickly slide to the left side of the page, rotating
+counterclockwise as they move and remaining in a vertical line.
+
+Once completed, add some transitions or translations to previous projects.
+
+Remember to keep them simple and subtle!
+ */
+
+.grid:hover .odd{
```

Comment:
I think your hover needs a transition duration. its changes instantly and does not do a smooth size and rotation change.